source/Halibut.Tests/Support/SerilogLoggerBuilder.cs

```diff
@@ -105,5 +115,35 @@ public void Emit(LogEvent logEvent)
                 }
             }
         }
+
+        public class TraceLogsForFailedTestsSink : ILogEventSink
+        {
+            private readonly MessageTemplateTextFormatter _formatter;
```

Comment:
Nit: I love a good underscore, and it's my preferred format. But I believe our house style is to not use the underscore.

Also, I think our house style is to drop the 'private' (again, I prefer to keep it, but it's not my choice 😞 )